foruma-frontend/src/stories/Page.stories.js

```diff
@@ -0,0 +1,29 @@
+import MyPage from './Page.vue';
+import * as HeaderStories from './Header.stories';
```

Comment:
Codacy has a fix for the issue: [Replace `'./Header.stories'` with `"./Header.stories"`](https://app.codacy.com/gh/agabani/foruma/pullRequest?prid=7388752)
```suggestion
import * as HeaderStories from "./Header.stories";
```